Auto update failing for zrok

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [zrok](https://github.com/termux/termux-packages/blob/ebe6a5d71a6b21686b17fbcaaf0355bcc27cdf56/packages/zrok) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating zrok [Current version: 0.4.49]
INFO: package being updated to 1.0.0.
Downloading https://github.com/openziti/zrok/archive/refs/tags/v1.0.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 8851k    0 8851k    0     0  8604k      0 --:--:--  0:00:01 --:--:-- 12.0M

You are about to commit these changes:

--------------------
diff --git a/packages/zrok/build.sh b/packages/zrok/build.sh
index 38e5150236..ee784faf85 100644
--- a/packages/zrok/build.sh
+++ b/packages/zrok/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://zrok.io/
 TERMUX_PKG_DESCRIPTION="An open source sharing solution built on OpenZiti."
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
-TERMUX_PKG_VERSION="0.4.49"
+TERMUX_PKG_VERSION="1.0.0"
 TERMUX_PKG_SRCURL=https://github.com/openziti/zrok/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=32dc578a4eb2b728cb378409b15d7e834a1677f0ea87fb1d029455983ddaab7c
+TERMUX_PKG_SHA256=40eeba493b0dd9b5ebf16e9a95209264e82c2e8b4bf0b615e0f1e5d53ebfaaae
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_AUTO_UPDATE=true
 
--------------------

bump(main/zrok): 1.0.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package zrok exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building zrok for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  45178      0 --:--:-- --:--:-- --:--:-- 45134
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1384      0 --:--:-- --:--:-- --:--:--  1386
gpg: Signature made Fri 14 Mar 2025 06:26:21 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1666k  100 1666k    0     0  2363k      0 --:--:-- --:--:-- --:--:-- 2363k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  24197      0 --:--:-- --:--:-- --:--:-- 24174
100 13320  100 13320    0     0  24192      0 --:--:-- --:--:-- --:--:-- 24174
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1549      0 --:--:-- --:--:-- --:--:--  1551
gpg: Signature made Fri 14 Mar 2025 12:26:56 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  23882      0 --:--:-- --:--:-- --:--:-- 23854
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1483      0 --:--:-- --:--:-- --:--:--  1484
gpg: Signature made Fri 14 Mar 2025 02:27:05 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://github.com/openziti/zrok/archive/refs/tags/v1.0.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 91501    0 91501    0     0   177k      0 --:--:-- --:--:-- --:--:--  177k
100 8851k    0 8851k    0     0  9760k      0 --:--:-- --:--:-- --:--:-- 21.2M
Downloading https://nodejs.org/dist/v20.17.0/node-v20.17.0-linux-x64.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 38 24.5M   38 9598k    0     0  22.8M      0  0:00:01 --:--:--  0:00:01 22.8M
100 24.5M  100 24.5M    0     0  34.0M      0 --:--:-- --:--:-- --:--:-- 34.0M
Downloading https://golang.org/dl/go1.24.1.linux-amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    80  100    80    0     0    690      0 --:--:-- --:--:-- --:--:--   695

100    75  100    75    0     0    277      0 --:--:-- --:--:-- --:--:--   277

 47 74.8M   47 35.2M    0     0  43.3M      0  0:00:01 --:--:--  0:00:01 43.3M
100 74.8M  100 74.8M    0     0  65.8M      0  0:00:01  0:00:01 --:--:--  122M
~/.termux-build/zrok/src/ui ~/.termux-build/zrok/src

added 315 packages, and audited 316 packages in 10s

69 packages are looking for funding
  run `npm fund` for details

2 moderate severity vulnerabilities

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
npm notice
npm notice New major version of npm available! 10.8.2 -> 11.2.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.2.0
npm notice To update run: npm install -g npm@11.2.0
npm notice

> ui100@0.0.0 build
> vite build

vite v6.2.0 building for production...
transforming...
✓ 2336 modules transformed.
rendering chunks...
computing gzip size...
dist/index.html                     0.89 kB │ gzip:   0.46 kB
dist/assets/index-BWg0ExTa.css     15.84 kB │ gzip:   2.82 kB
dist/assets/index-25c3sj4i.js   1,493.07 kB │ gzip: 455.63 kB

(!) Some chunks are larger than 500 kB after minification. Consider:
- Using dynamic import() to code-split the application
- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-options/#output-manualchunks
- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
✓ built in 6.87s
~/.termux-build/zrok/src
go: warning: ignoring go.mod in $GOPATH /home/builder/.termux-build/zrok/src
go: downloading github.com/sirupsen/logrus v1.9.3
go: downloading github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.0
go: downloading github.com/kolesnikovae/go-winjob v1.0.0
go: downloading google.golang.org/genproto/googleapis/api v0.0.0-20240814211410-ddb44dafa142
go: downloading google.golang.org/grpc v1.67.1
go: downloading github.com/michaelquigley/pfxlog v0.6.10
go: downloading github.com/pkg/errors v0.9.1
go: downloading google.golang.org/protobuf v1.34.2
go: downloading github.com/openziti/sdk-golang v0.23.44
go: downloading github.com/michaelquigley/cf v0.0.13
go: downloading github.com/influxdata/influxdb-client-go/v2 v2.11.0
go: downloading github.com/jmoiron/sqlx v1.3.5
go: downloading github.com/openziti/edge-api v0.26.34
go: downloading github.com/wneessen/go-mail v0.2.7
go: downloading github.com/go-openapi/errors v0.22.0
go: downloading github.com/go-openapi/loads v0.22.0
go: downloading github.com/go-openapi/runtime v0.28.0
go: downloading github.com/jaevor/go-nanoid v1.3.0
go: downloading github.com/jessevdk/go-flags v1.6.1
go: downloading github.com/lib/pq v1.10.9
go: downloading github.com/mattn/go-sqlite3 v1.14.18
go: downloading github.com/gorilla/websocket v1.5.3
go: downloading golang.org/x/crypto v0.35.0
go: downloading github.com/nxadm/tail v1.4.8
go: downloading github.com/openziti/channel/v3 v3.0.6
go: downloading github.com/openziti/identity v1.0.86
go: downloading github.com/openziti/ziti v1.1.15
go: downloading github.com/rabbitmq/amqp091-go v1.8.1
go: downloading github.com/iancoleman/strcase v0.3.0
go: downloading github.com/rubenv/sql-migrate v1.6.0
go: downloading github.com/charmbracelet/bubbles v0.14.0
go: downloading github.com/charmbracelet/bubbletea v0.23.1
go: downloading github.com/caddyserver/caddy/v2 v2.8.4
go: downloading github.com/charmbracelet/lipgloss v0.6.0
go: downloading github.com/gobwas/glob v0.2.3
go: downloading github.com/golang-jwt/jwt/v5 v5.2.1
go: downloading github.com/jedib0t/go-pretty/v6 v6.5.9
go: downloading github.com/google/uuid v1.6.0
go: downloading github.com/muesli/reflow v0.3.0
go: downloading github.com/zitadel/oidc/v2 v2.12.2
go: downloading github.com/openziti/cobra-to-md v1.0.1
go: downloading github.com/openziti/transport/v2 v2.0.147
go: downloading golang.org/x/oauth2 v0.23.0
go: downloading github.com/shirou/gopsutil/v3 v3.24.5
go: downloading github.com/spf13/cobra v1.8.1
go: downloading github.com/google/go-cmp v0.6.0
go: downloading golang.org/x/time v0.6.0
go: downloading github.com/net-byte/vtun v1.7.0
go: downloading nhooyr.io/websocket v1.8.17
go: downloading github.com/net-byte/water v0.0.7
go: downloading github.com/greenpau/caddy-security v1.1.29
go: downloading github.com/go-openapi/strfmt v0.23.0
go: downloading github.com/orcaman/concurrent-map/v2 v2.0.1
go: downloading go.uber.org/zap v1.27.0
go: downloading github.com/songgao/water v0.0.0-20200317203138-2b4b6d7c09d8
go: downloading github.com/go-openapi/swag v0.23.0
go: downloading github.com/go-openapi/validate v0.24.0
go: downloading golang.org/x/net v0.29.0
go: downloading github.com/go-openapi/spec v0.21.0
go: downloading github.com/TwiN/go-away v1.6.12
go: downloading golang.org/x/sys v0.30.0
go: downloading github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d
go: downloading google.golang.org/genproto/googleapis/rpc v0.0.0-20240814211410-ddb44dafa142
go: downloading github.com/cenkalti/backoff/v4 v4.3.0
go: downloading github.com/kataras/go-events v0.0.3
go: downloading github.com/openziti/foundation/v2 v2.0.49
go: downloading github.com/openziti/metrics v1.2.58
go: downloading github.com/openziti/secretstream v0.1.25
go: downloading github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
go: downloading github.com/mitchellh/mapstructure v1.5.0
go: downloading golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56
go: downloading gopkg.in/yaml.v3 v3.0.1
go: downloading github.com/go-openapi/analysis v0.23.0
go: downloading github.com/fullsailor/pkcs7 v0.0.0-20190404230743-d7302db945fa
go: downloading github.com/influxdata/line-protocol v0.0.0-20200327222509-2487e7298839
go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: downloading github.com/Jeffail/gabs v1.4.0
go: downloading github.com/fsnotify/fsnotify v1.7.0
go: downloading github.com/go-gorp/gorp/v3 v3.1.0
go: downloading github.com/atotto/clipboard v0.1.4
go: downloading github.com/mattn/go-runewidth v0.0.15
go: downloading github.com/containerd/console v1.0.3
go: downloading github.com/mattn/go-isatty v0.0.20
go: downloading github.com/mattn/go-localereader v0.0.1
go: downloading github.com/muesli/ansi v0.0.0-20211018074035-2e021307bc4b
go: downloading github.com/muesli/cancelreader v0.2.2
go: downloading github.com/muesli/termenv v0.13.0
go: downloading golang.org/x/term v0.29.0
go: downloading github.com/caddyserver/certmagic v0.21.3
go: downloading github.com/cespare/xxhash/v2 v2.3.0
go: downloading github.com/prometheus/client_golang v1.19.1
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/quic-go/quic-go v0.44.0
go: downloading go.uber.org/zap/exp v0.2.0
go: downloading golang.org/x/sync v0.11.0
go: downloading github.com/opentracing/opentracing-go v1.2.1-0.20220228012449-10b1cf09e00b
go: downloading go.opentelemetry.io/otel v1.30.0
go: downloading go.opentelemetry.io/otel/trace v1.30.0
go: downloading gopkg.in/go-jose/go-jose.v2 v2.6.3
go: downloading github.com/gorilla/securecookie v1.1.2
go: downloading github.com/gorilla/schema v1.4.1
go: downloading github.com/muhlemmer/gu v0.3.1
go: downloading golang.org/x/text v0.22.0
go: downloading github.com/openziti/storage v0.3.2
go: downloading cloud.google.com/go/compute/metadata v0.5.0
go: downloading github.com/dustin/go-humanize v1.0.1
go: downloading github.com/mholt/acmez/v2 v2.0.1
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/google/cel-go v0.20.1
go: downloading github.com/oklog/ulid v1.3.1
go: downloading github.com/asaskevich/govalidator v0.0.0-20230301143203-a9d515a09cc2
go: downloading go.mongodb.org/mongo-driver v1.17.0
go: downloading github.com/greenpau/go-authcrunch v1.1.4
go: downloading go.uber.org/multierr v1.11.0
go: downloading github.com/mailru/easyjson v0.7.7
go: downloading github.com/docker/go-units v0.5.0
go: downloading github.com/go-openapi/jsonpointer v0.21.0
go: downloading github.com/go-openapi/jsonreference v0.21.0
go: downloading github.com/mattn/go-colorable v0.1.13
go: downloading github.com/go-resty/resty/v2 v2.15.3
go: downloading github.com/mitchellh/go-ps v1.0.0
go: downloading github.com/speps/go-hashids v2.0.0+incompatible
go: downloading github.com/golang/protobuf v1.5.4
go: downloading github.com/deepmap/oapi-codegen v1.8.2
go: downloading github.com/rivo/uniseg v0.4.4
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/prometheus/client_model v0.5.0
go: downloading github.com/prometheus/common v0.48.0
go: downloading github.com/prometheus/procfs v0.12.0
go: downloading github.com/aymanbagabas/go-osc52 v1.0.3
go: downloading github.com/lucasb-eyer/go-colorful v1.2.0
go: downloading github.com/smallstep/certificates v0.26.1
go: downloading github.com/smallstep/truststore v0.13.0
go: downloading go.step.sm/crypto v0.45.0
go: downloading github.com/go-chi/chi/v5 v5.0.12
go: downloading github.com/smallstep/nosql v0.6.1
go: downloading github.com/caddyserver/zerossl v0.1.3
go: downloading github.com/klauspost/cpuid/v2 v2.2.7
go: downloading github.com/tailscale/tscert v0.0.0-20240517230440-bbccfbf48933
go: downloading gopkg.in/natefinch/lumberjack.v2 v2.2.1
go: downloading github.com/libdns/libdns v0.2.2
go: downloading github.com/miekg/dns v1.1.62
go: downloading github.com/zeebo/blake3 v0.2.3
go: downloading github.com/quic-go/qpack v0.4.0
go: downloading github.com/go-logr/logr v1.4.2
go: downloading go.opentelemetry.io/otel/metric v1.30.0
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.4
go: downloading github.com/aryann/difflib v0.0.0-20210328193216-ff5ff6dc229b
go: downloading go.uber.org/automaxprocs v1.5.3
go: downloading golang.org/x/crypto/x509roots/fallback v0.0.0-20240507223354-67b13616a595
go: downloading github.com/biogo/store v0.0.0-20200525035639-8c94ae1e7c9c
go: downloading github.com/stretchr/testify v1.9.0
go: downloading go.etcd.io/bbolt v1.3.11
go: downloading github.com/stoewer/go-strcase v1.3.0
go: downloading github.com/antlr4-go/antlr/v4 v4.13.1
go: downloading github.com/BurntSushi/toml v1.3.2
go: downloading github.com/Masterminds/sprig/v3 v3.2.3
go: downloading github.com/alecthomas/chroma/v2 v2.13.0
go: downloading github.com/yuin/goldmark v1.7.1
go: downloading github.com/yuin/goldmark-highlighting/v2 v2.0.0-20230729083705-37449abec8cc
go: downloading github.com/gobwas/ws v1.2.1
go: downloading github.com/net-byte/go-gateway v0.0.2
go: downloading github.com/emersion/go-sasl v0.0.0-20231106173351-e73c9f7bad43
go: downloading github.com/emersion/go-smtp v0.21.0
go: downloading github.com/skip2/go-qrcode v0.0.0-20200617195104-da1b6568686e
go: downloading github.com/josharian/intern v1.0.0
go: downloading github.com/tklauser/go-sysconf v0.3.14
go: downloading github.com/parallaxsecond/parsec-client-go v0.0.0-20221025095442-f0a77d263cf9
go: downloading github.com/klauspost/compress v1.17.8
go: downloading github.com/pires/go-proxyproto v0.7.0
go: downloading go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.49.0
go: downloading go.opentelemetry.io/contrib/propagators/autoprop v0.42.0
go: downloading go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.21.0
go: downloading go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.21.0
go: downloading go.opentelemetry.io/otel/sdk v1.28.0
go: downloading go.step.sm/cli-utils v0.9.0
go: downloading go.step.sm/linkedca v0.20.1
go: downloading github.com/fxamacker/cbor/v2 v2.6.0
go: downloading github.com/google/go-tpm v0.9.0
go: downloading github.com/smallstep/go-attestation v0.4.4-0.20240109183208-413678f90935
go: downloading github.com/slackhq/nebula v1.7.2
go: downloading github.com/go-logr/stdr v1.2.2
go: downloading github.com/russross/blackfriday/v2 v2.1.0
go: downloading github.com/Masterminds/goutils v1.1.1
go: downloading github.com/Masterminds/semver/v3 v3.2.1
go: downloading github.com/huandu/xstrings v1.4.0
go: downloading github.com/imdario/mergo v0.3.16
go: downloading github.com/mitchellh/copystructure v1.2.0
go: downloading github.com/shopspring/decimal v1.3.1
go: downloading github.com/spf13/cast v1.6.0
go: downloading github.com/gobwas/httphead v0.1.0
go: downloading github.com/gobwas/pool v0.2.1
go: downloading github.com/inhies/go-bytesize v0.0.0-20210819104631-275770b98743
go: downloading github.com/golang-jwt/jwt/v4 v4.5.1
go: downloading github.com/greenpau/versioned v1.0.30
go: downloading github.com/tklauser/numcpus v0.8.0
go: downloading github.com/felixge/httpsnoop v1.0.4
go: downloading go.opentelemetry.io/proto/otlp v1.0.0
go: downloading github.com/smallstep/pkcs7 v0.0.0-20231107075624-be1870d87d13
go: downloading github.com/smallstep/scep v0.0.0-20231024192529-aee96d7ad34d
go: downloading go.opentelemetry.io/contrib/propagators/aws v1.17.0
go: downloading go.opentelemetry.io/contrib/propagators/b3 v1.17.0
go: downloading go.opentelemetry.io/contrib/propagators/jaeger v1.17.0
go: downloading go.opentelemetry.io/contrib/propagators/ot v1.17.0
go: downloading github.com/urfave/cli v1.22.14
go: downloading github.com/go-jose/go-jose/v3 v3.0.3
go: downloading github.com/chzyer/readline v1.5.1
go: downloading github.com/manifoldco/promptui v0.9.0
go: downloading github.com/x448/float16 v0.8.4
go: downloading github.com/google/go-tspi v0.3.0
go: downloading github.com/rs/xid v1.5.0
go: downloading filippo.io/edwards25519 v1.1.0
go: downloading github.com/dgraph-io/badger v1.6.2
go: downloading github.com/dgraph-io/badger/v2 v2.2007.4
go: downloading github.com/go-sql-driver/mysql v1.7.1
go: downloading github.com/jackc/pgx/v4 v4.18.3
go: downloading github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
go: downloading github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
go: downloading github.com/mitchellh/reflectwalk v1.0.2
go: downloading github.com/dlclark/regexp2 v1.11.0
go: downloading github.com/crewjam/saml v0.4.14
go: downloading github.com/go-ldap/ldap/v3 v3.4.7
go: downloading github.com/go-kit/kit v0.13.0
go: downloading github.com/google/certificate-transparency-go v1.1.8-0.20240110162603-74a5dd331745
go: downloading github.com/dgraph-io/ristretto v0.1.1
go: downloading github.com/dgryski/go-farm v0.0.0-20200201041132-a6ae2369ad13
go: downloading github.com/jackc/pgconn v1.14.3
go: downloading github.com/jackc/pgio v1.0.0
go: downloading github.com/jackc/pgtype v1.14.0
go: downloading github.com/jackc/pgproto3/v2 v2.3.3
go: downloading github.com/beevik/etree v1.3.0
go: downloading github.com/mattermost/xml-roundtrip-validator v0.1.0
go: downloading github.com/russellhaering/goxmldsig v1.4.0
go: downloading github.com/crewjam/httperr v0.2.0
go: downloading github.com/go-kit/log v0.2.1
go: downloading github.com/AndreasBriese/bbloom v0.0.0-20190825152654-46b345b51c96
go: downloading github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358
go: downloading github.com/go-asn1-ber/asn1-ber v1.5.5
go: downloading github.com/golang/snappy v0.0.4
go: downloading github.com/golang/glog v1.2.2
go: downloading github.com/jackc/chunkreader/v2 v2.0.1
go: downloading github.com/jackc/pgpassfile v1.0.0
go: downloading github.com/jackc/pgservicefile v0.0.0-20231201235250-de7065d80cb9
go: downloading github.com/jonboulle/clockwork v0.4.0
go: downloading github.com/go-logfmt/logfmt v0.6.0
go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
agent/agentUi/embed.go:5:12: pattern dist: no matching files found
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-03-14 18:37:37 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/13862628464">13862628464</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/zrok/build.sh
+++ b/packages/zrok/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://zrok.io/
 TERMUX_PKG_DESCRIPTION="An open source sharing solution built on OpenZiti."
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
-TERMUX_PKG_VERSION="0.4.49"
+TERMUX_PKG_VERSION="1.0.0"
 TERMUX_PKG_SRCURL=https://github.com/openziti/zrok/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=32dc578a4eb2b728cb378409b15d7e834a1677f0ea87fb1d029455983ddaab7c
+TERMUX_PKG_SHA256=40eeba493b0dd9b5ebf16e9a95209264e82c2e8b4bf0b615e0f1e5d53ebfaaae
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_AUTO_UPDATE=true
 
@@ -16,11 +16,13 @@ termux_step_make() {
 	termux_setup_nodejs
 	termux_setup_golang
 
-	pushd ui
-	cd "$TERMUX_PKG_SRCDIR/ui"
-	npm install
-	npm run build
-	popd
+	dirs=("ui" "agent/agentUi")
+	for dir in "${dirs[@]}"; do
+		pushd "$dir"
+		npm install
+		npm run build
+		popd
+	done
 
 	mkdir -p  "$TERMUX_PKG_SRCDIR/dist"
 

